docs/cheatsheet.md

```diff
@@ -0,0 +1,168 @@
+// once done execute start line and check it:
+torquser@homer:/home/torquser/newdeploy$ ./deploy/bin/torq.sh start all
+08:28:02 | Starting discovery1...
+08:28:02 | Starting tickerplant1...
+... snip ...
+08:28:06 | Starting dqe1...
+08:28:06 | Starting dqedb1...
+torquser@homer:/home/torquser/newdeploy$ ./deploy/bin/torq.sh summary
+TIME      |  PROCESS        |  STATUS  |  PID    |  PORT
+08:28:30  |  discovery1     |  up      |  23221  |  43101
+08:28:30  |  tickerplant1   |  up      |  23333  |  43100
+08:28:30  |  rdb1           |  up      |  23443  |  43102
+08:28:30  |  hdb1           |  up      |  23553  |  43103
+... snip ...
+```
+
+## Modifying Existing Installations
+We always try to esnure new versions of TorQ are backwardly compatible. Try to avoid modifying TorQ itself and instead make application specific modifications. The installation script deploys TorQ in a structure which keeps TorQ and the application separate, and this should be adhered to whenever possible. 
+
+To modify the config for a process, do it either in the [config directory](http://aquaqanalytics.github.io/TorQ/gettingstarted/#configuration-loading) or as a command line start up parameter as- almost all config variables that are defined in a config file and exists in a non-root namespace can be overridden from the command line. 
+
+To make a process load additional files, you can:
+
+- append the additional files to the -load parameter on the start line
+- append a set of directories of files using -loaddir on the start line
+- place the files in one of the directories that is [loaded by default on start up](http://aquaqanalytics.github.io/TorQ/gettingstarted/#code-loading)
+
+The latter is probably preferable. 
+
+## Adding Custom Processes
+
+When adding custom processes to TorQ it is important to [understand the significance of proctype](http://aquaqanalytics.github.io/TorQ/gettingstarted/#process-identification). In a nutshell, only processes which do exactly the same thing should have the same proctype. If two processes do roughly the same thing then -parentproctype can be used to share common functionality. Also there isn't any formal association between proctype and code name file name e.g. a process that loads code/processes/rdb.q can have any proctype we like. 
+
+How TorQ [manages connections](http://aquaqanalytics.github.io/TorQ/conn/) is important. Avoid using hopen, use TorQ connection management.
+
+TorQ uses the fail fast principle (if you are going to fail, may as well do it as quickly as possible). This helps avoid processes starting up in inconsistent or unexpected states. If running a process with the -debug option, add the -stop or -trap options to stop at, or trap and continue through, start up errors. 
+
+Code is loaded in a specific order, which can be overridden. To determine the order, inspect the bottom of the torq.q script (the last 50 lines or so).
```

Comment:
This is a little bit like throwing a dart at a map, perhaps there is some specific function(s) that could be referenced to help find this section of `torq.q`? Obviously line numbers could change over time and that won't work.